mitodl/ol-keycloak
# Switched UI over to using fullname

### What are the relevant tickets?




Part of https://github.com/mitodl/hq/issues/5559

### Description (What does it do?)

This switches the UX on registration to only show the full name instead of first/last names.

Related PR in infrastructure that adds configuration for the attribute changes: https://github.com/mitodl/ol-infrastructure/pull/2685

### Screenshots (if appropriate):

![Screenshot 2024-09-26 at 01-29-23 MIT Learn](https://github.com/user-attachments/assets/d105b661-9c1d-4500-b481-812ab82014f9)
![Screenshot 2024-09-25 at 16-43-35 MIT Learn](https://github.com/user-attachments/assets/d2f4c45e-221f-44bf-836a-3028c6166e2f)


### How can this be tested?

- In the realm admin under "User Profile":
  - Make first and last names optional
  - Add a `lastName` attribute that is required
- Verify that for existing users, you're prompted to enter your full name when logging in
- Verify that when registering you're only asked for full name and can successfully register.
